distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -53,12 +56,16 @@ grant
     // some permissions might be repeated here to ensure that they are granted to the migration
     // framework if we ever tighten the permissions above
 
+    permission javax.management.MBeanServerPermission "createMBeanServer";
+
     permission java.io.FilePermission "<<ALL FILES>>", "read, readlink";
     permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}-", "read, write, delete";
```

Comment:
✏️ This change is likely to have huge repercussions and will require a full regression pass. I'm particularly worried about blocking the `delete` permission on files beneath `$ddf.home` globally without granting that permission to several specific `codeBase`s.

Reply:
This permission is granted only to the migration framework.

Reply:
I read that diff wrong and thought that was in the global deny block. My mistake.

Reply:
I would rather we discuss it now then later when it is too late ;-)